internal/access/access_test.go

```diff
@@ -86,11 +86,11 @@ func TestBasicGrant(t *testing.T) {
 	cant(t, db, "i:bob", "read", "infra.groups.1") // currently we check for exact grant match, this may change as grants evolve
 	cant(t, db, "i:bob", "write", "infra.groups")
 
-	grant(t, db, tom, "i:alice", "read", "infra.machines")
-	can(t, db, "i:alice", "read", "infra.machines")
-	cant(t, db, "i:alice", "read", "infra")
-	cant(t, db, "i:alice", "read", "infra.machines.1")
-	cant(t, db, "i:alice", "write", "infra.machines")
+	grant(t, db, tom, "i:a11ce", "read", "infra.machines")
```

Comment:
These IDs had to change now that `data.ListGrants` checks if the ID is a valid userID.